docker/openproblems-r-extras/README.md

```diff
@@ -13,6 +13,8 @@ R packages:
 * batchelor
 * sparsesvd
 * dplyr
+* RCTD
```

Comment:
Renamed to spacexr

Reply:
I'm not sure whether we can/should adress this @almaan what do you think?

Reply:
Solution here is just to delete RCTD and add spacexr -- they renamed the package.